change/@ni-nimble-components-8c244660-446c-4db7-bc74-430a876bae30.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Adding tests.",
```

Comment:
nit: Could be a tad more specific so the chagelog entry is more self descriptive 'Adding table string value and row / column element tests'

Reply:
While you're at it, please clean up the PR title too (could match Milan's suggestion for the change log comment).